### DISCUSSION TITLE
AXN East Asia (Vietnamese)

### DISCUSSION THREAD

**Discussion by jdkeloth** [iptv-org/community#684] [Q&A]
**_Channel Name:_** AXN East Asia(Vietnam)
**_Channel ID:_** AXN.vn
**_Broken Link (from playlist):_** Vietnam
**_Possible Replacement (optional):_** xxx
**_Notes (optional):_** Stream is not working. Was working yesterday but is all of a sudden down today. Could you have a look at a stream and see if you can fix it? 🙏 

Regards 
Keloth Justyn Dylan

**Comment by Dum4G** [comment]
All fptplay links need to be checked. The website itself greets me with Error 500 and any links from there are forbidden even with user-agent we used before

**Comment by Dum4G** [comment]
@LaneSh4d0w

**Comment by BellezaEmporium** [comment]
Having the same issues too. Gonna see if I can have some stuff tomorrow.

**Comment by Dum4G** [comment]
It's only AXN and FBNC with those odd old and complicated links. I tried everything with user-agent and referrer, only makes it to connect longer

**Comment by Dum4G** [comment]
Just as I said that I took FBNC under my control. No luck for AXN :(

**Comment by BellezaEmporium** [comment]
All the other Vietnamese links seems to have locked down too. Same issue than AXN.

**Comment by Dum4G** [comment]
Do they? I accessed one just fine. Anyway, you can update user-agent to something up-to-date and it will be working again

**Comment by BellezaEmporium** [comment]
> 
> 
> Do they? I accessed one just fine. Anyway, you can update user-agent to something up-to-date and it will be working again

Unless they have the "Chinese browser user-agent block" issue, there's saying Forbidden on my side. Gonna check.

**Comment by MinePlayersPE** [comment]
Note: https://github.com/iptv-org/iptv/discussions/3303 means that the above link can't be added to this repo
also mncnow is indonesian

**Comment by BellezaEmporium** [comment]
Only Kodi to this date is able to take care of DRM-locked channels.

**Comment by MinePlayersPE** [comment]
>Only Kodi to this date is able to take care of DRM-locked channels.

@LaneSh4d0w Doesn't seem to be true anymore, see https://github.com/thisistyx/unifi-tv#players-that-can-play-this-playlist which uses Widevine-encrypted streams

**Comment by BellezaEmporium** [comment]
I do have the native mpeg/dash plugin (thanks to @jkemp814 who tipped me this extension). I'm gonna have a look at it.
